### DISCUSSION TITLE
Mimic Clickhouse queries' numbers to numbers reported in dashboard

### DISCUSSION THREAD

**Discussion by measurestrategy** [ClickHouse/ClickHouse#70528] [Q&A]
Hi, thanks in advance for the help.
I'm using Plausible Community Edition and I would like to run some queries directly in Plausible's Clickhouse db to automate some data extractions so trying basic queries to check first place if numbers are matching what we see in the dashboard and avoid the classic "numbers don't match!"

For the exercise I select a date range 2 Oct to 6 Oct
GMT+00:00 UTC in time zone configuration because timestamps are “absolute time”, or in another words, assume “always UTC”
interface reports **1,834,947** unique visitors.

<img width="1002" alt="plau" src="https://github.com/user-attachments/assets/27c7efe4-7672-468b-8c2d-d9bab25b1570">

This query reports **1,825,034**
```
SELECT COUNT(DISTINCT user_id) as unique_visitors
FROM "plausible_events_db"."sessions_v2"
WHERE timestamp BETWEEN '2024-10-02 00:00:00' AND '2024-10-06 23:59:59'
```
In all fairness it is just **-0.54%** difference between results but is there anything I'm missing in my query that could make it match the dashboard?

**Comment by Algunenano** [comment]
This question should be first directed towards Plausible. We have no good way of knowing which queries or data is being used to show these values and whether they should match or not.

**Reply by measurestrategy** [reply]
My bad, Raúl, apologies.
I thought I was on Plausible > Clickhouse support.
